src/orca/services/sevenbridges/models.py

```diff
@@ -0,0 +1,23 @@
+from pydantic import validator
+from pydantic.dataclasses import dataclass
+from sevenbridges.models.enums import TaskStatus as TaskState
+
+
+@dataclass(kw_only=False)
+class TaskStatus:
```

Comment:
Probably not important now, but once we add more connectors, it could make sense to have a base status class.

Reply:
Agreed. We'll just have to think of how to handle the fact that the `state` attribute will consist of different enums.